sdk/include/reactos/undocuser.h

```diff
@@ -204,6 +204,20 @@ BOOL WINAPI PaintMenuBar(HWND hWnd, HDC hDC, ULONG left, ULONG right, ULONG top,
 #define DrawCaptionTemp DrawCaptionTempA
 #endif
 
+//
+// Hard error balloon package
+//
+typedef struct _BALLOON_HARD_ERROR_DATA
+{
+    DWORD cbHeaderSize;
+    DWORD Status;
+    DWORD dwResponseType;
+    ULONG_PTR TitleOffset;
```

Comment:
Wrong! This is ULONG/DWORD *even* in x64! Otherwise the code doesn't work on Win7 x64 nor on Win2003 x64.

Reply:
Please check this on your Windows before committing :)